recipes/serialio/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "serialio" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/serialio-{{ version }}.tar.gz
+  sha256: b3a4e2e173b521ae54b70995c88a01be50bab9cc718c05da89857d4ce480aaff
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - pyserial >=3
+    - sockio >=0.15
+
+test:
+  imports:
+    - serialio
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/tiagocoutinho/serialio
+  summary: Concurrency agnostic serialio API
+  license: GPL-3.0-or-later
```

Comment:
A different license is mentioned upstream. Can you please clarify which one it is?

Ref: License :: OSI Approved :: GNU Lesser General Public License v2 or later (LGPLv2+)